recipes/aiida-core/conda_build_config.yaml

```diff
@@ -0,0 +1,6 @@
+channel_sources:
+ - defaults
+ - conda-forge
+ - etetoolkit
```

Comment:
what packages are being pulled from etetoolkit?

Reply:
We wanted to pull the `ete3` package from `etetoolkit` but for some reason this did not work (see [travis log](https://travis-ci.org/conda-forge/staged-recipes/builds/466407343#L359) ).
Do you happen to know why? Are the `channel_sources` actually being processed?

Anyhow, for the moment, we simply removed the `ete3` dependency since it is only used in one place in aiida-core.

Please note that there is an [open issue](https://github.com/conda-forge/staged-recipes/pull/5885) on adding ete3 to conda-forge but nobody replied to my offer to help with this for more than a month.
Still happy to help with this.

Reply:
You could open another PR if you want, it seems that the second to last comment shows some differences between the proposed and bioconda recipes. I'm not certain if that is an issue.

Reply:
Me neither. I'll try once more to get a response from the proposed maintainer

Reply:
Following the original PR https://github.com/conda-forge/staged-recipes/pull/5885, the `ete3` recipe [has `noarch: python`](https://github.com/conda-forge/staged-recipes/pull/7315/files#diff-85abb5325bb753d20fd87b98fe51d4d5R17).
After re-adding the `ete3` dependency below, the builds on all platforms fail because the `ete3` dependency on the platform is missing.

I can reproduce this locally:
```
$ conda search -c conda-forge ete3 --platform osx-64
Loading channels: done

PackagesNotFoundError: The following packages are not available from current channels:

  - ete3
...
```

I thought `noarch` would mean it should be compatible with all architectures. Can you tell me what I'm overlooking here?